src/plrmodes/marisa.c

```diff
@@ -21,8 +21,8 @@ PlayerCharacter character_marisa = {
 	.title = "Unbelievably Ordinary Magician",
 	.menu_texture_name = "marisa_bombbg",
 	.ending = {
-		.good = good_ending_marisa,
-		.bad = bad_ending_marisa,
+		.good = CUTSCENE_ID_MARISA_GOOD_END, ENDING_GOOD_MARISA,
+		.bad = CUTSCENE_ID_MARISA_BAD_END, ENDING_BAD_MARISA,
```

Comment:
I'd prefer to have these wrapped in braces for clarity; same for the other characters.

```suggestion
		.good = { CUTSCENE_ID_MARISA_GOOD_END, ENDING_GOOD_MARISA },
		.bad = { CUTSCENE_ID_MARISA_BAD_END, ENDING_BAD_MARISA },
```